src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlUdtInfo.cs

```diff
@@ -32,7 +35,7 @@ internal static SqlUdtInfo GetFromType(Type target)
             SqlUdtInfo udtAttr = TryGetFromType(target);
             if (udtAttr == null)
             {
-                throw InvalidUdtException.Create(target, Strings.SqlUdtReason_NoUdtAttribute);
+                throw Server.InvalidUdtException.Create(target, Strings.SqlUdtReason_NoUdtAttribute);
```

Comment:
If InvalidUstException bubbles up to users, this is another one we need to move to and get from Microsoft.SqlServer.Server. That's where it was originally:

https://github.com/dotnet/corefx/blob/release/3.1/src/System.Data.SqlClient/src/System/Data/Sql/InvalidUdtException.cs